src/System.Text.Json/src/System/Text/Json/Serialization/JsonNameAttribute.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Text.Json.Serialization
+{
+    /// <summary>
+    /// Specifies a property name.
```

Comment:
Should add that `JsonNameAttribute` will override the name policy.

Reply:
Done. Thanks.